src/main/java/Dude.java

```diff
@@ -0,0 +1,122 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Dude {
+
+//    static Task[] taskList = new Task[100];
+    static ArrayList<Task> taskList = new ArrayList<Task>();
+    static int nTasks = 0;
+
+    public static void addTodo(String task) {
+        ToDo newTask = new ToDo(task);
+//        taskList[nTasks] = newTask;
+        taskList.add(newTask);
+        System.out.printf("Got it. I've added this task:\n%s\n", newTask.toString());
+        nTasks += 1;
+        System.out.printf("Now you have %d tasks in the list. \n", nTasks);
+    }
+    public static void addDeadline(String task, String by) {
+        Deadline newTask = new Deadline(task, by);
+//        taskList[nTasks] = newTask;
+        taskList.add(newTask);
+        System.out.printf("Got it. I've added this task:\n%s\n", newTask.toString());
+        nTasks += 1;
+        System.out.printf("Now you have %d tasks in the list. \n", nTasks);
+    }
+    public static void addEvent(String task, String from, String to) {
+        Event newTask = new Event(task, from, to);
+//        taskList[nTasks] = newTask;
+        taskList.add(newTask);
+        System.out.printf("Got it. I've added this task:\n%s\n", newTask.toString());
+        nTasks += 1;
+        System.out.printf("Now you have %d tasks in the list. \n", nTasks);
+    }
+
+    public static void delete(int n) {
+        Task removedTask = taskList.get(n-1);
+        taskList.remove(n-1);
+        nTasks -= 1;
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(removedTask.toString());
+        System.out.printf("Now you have %d tasks in the list.\n", nTasks);
+
+    }
+
+    public static void list() {
+        for (int i = 0; i < nTasks; i++) {
+            Task task = taskList.get(i);
+//            Task task = taskList[i];
```

Comment:
You are using comments to temporarily remove code, however comments should be used to describe the code to the reader. They should also be indented to the relative position to your code i.e. the ```//``` needs to be on the same line as ```Task``` on line 47